cli/dcnios-cli.py

```diff
@@ -26,116 +26,150 @@
 import env
 import argparse
 
-types = ["dCache", "OSCAR", "S3", "SQS", "generic", "Kafka"]
-typesSSL = ["Kafka"]
+#types = ["dCache", "OSCAR", "S3", "SQS", "generic", "Kafka"]
+#typesSSL = ["Kafka"]
 
-def doType(type, function):
-    if type in data["nifi"]:
-        for singularoftype in data["nifi"][type]:
-            function(singularoftype["name"])
-            print(str(function.__qualname__) + " " + singularoftype["name"])
+"""def doType(type, function):
+    if type in data[env.NIFI_TAG]:
+        for singularoftype in data[env.NIFI_TAG][type]:
+            function(singularoftype[env.NAME_TAG])
+            print(str(function.__qualname__) + " " + singularoftype[env.NAME_TAG])
 
 
 def makeActionWithAllType(allType, function):
     for type in allType:
-        doType(type, function)
+        doType(type, function)"""
 
 def functionAll(function):
     for dic in env.info:
-        if dic["id"] in data["nifi"]:
-            for information in data["nifi"][dic["id"]]:
-                function(information["name"])
-                print(str(function.__qualname__) + " " + information["name"])
-                if "alterations" in information:
-                    for alter in information["alterations"]:
-                        function(alteration.nameActionReturn(alter["action"],information["name"]))
-                        print(str(function.__qualname__) + " " + alteration.nameActionReturn(alter["action"],information["name"]))
+        if dic["id"] in data[env.NIFI_TAG][dic["type"]]:
```

Comment:
There is too many nested code... this is going to require a pair programming session.

Reply:
Please, create a ticket to improve this part of the code.